completions/bd.fish

```diff
@@ -12,7 +12,7 @@ complete -c bd -s h -x --description "Display help and exit"
 complete -c bd -A -f
 
 function __fish_bd_complete_dirs
-	printf (pwd | sed 's|/|\\\n|g')
+    printf (pwd | sed 's%/[^/]*$%/%' | sed 's/.$//' | sed 's|/|\\\n|g')
```

Comment:
Could you please merge all those call to sed to a single call by merging all substitute commands together ?

Here we could just put the following:
```
printf (pwd | gsed 's%/[^/]*$%/%; s/.$//; s|/|\\\n|g')
```

Reply:
Thanks for your comments. That will be more concise. I have amended the commit as your suggestion.
In addition, I also recommend to hide the output of `pwd` and `echo "$newpwd"`, because most of the fish prompt have been able to show the path.